tests/vmi_cloudinit_test.go

```diff
@@ -539,47 +535,44 @@ var _ = Describe("[rfe_id:151][crit:high][vendor:cnv-qe@redhat.com][level:compon
 
 				By("checking cloudinit network-config")
 				CheckCloudInitFile(vmi, "openstack/latest/network_data.json", testNetworkData)
+				CheckCloudInitFile(vmi, "openstack/latest/user_data", testUserData)
```

Comment:
`user_data` does not have a file suffix?

Reply:
Seems not, according to other places in the code. Also, it passed ci